nebu/tests/cli/test_validate.py

```diff
@@ -11,8 +11,9 @@ def test_in_cwd(self, datadir, monkeypatch, invoker):
         assert result.exit_code == 0
         assert result.output == 'All good! :)\n'
 
-    def test_outside_cwd(self, datadir, invoker):
+    def test_outside_cwd(self, datadir, monkeypatch, invoker):
```

Comment:
When I looked in the code I could not find where `monkeypatch` comes (didn't find it in the fixtures). I wonder where it comes from. Just wanted to mention it.